srcpkgs/direwolf/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'direwolf'
+pkgname=direwolf
+version=1.6
+revision=1
+build_style=cmake
+configure_args="-Wno-dev"
+hostmakedepends="gcc make hamlib"
+makedepends="alsa-lib-devel libudev-devel hamlib-devel"
+depends="hamlib"
+short_desc="Software AX.25 TNC"
+maintainer="David Käppeler <d.kaeppeler@gmx.de>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/wb2osz/direwolf"
+distfiles="https://github.com/wb2osz/direwolf/archive/${version}.tar.gz"
+checksum=208b0563c9b339cbeb0e1feb52dc18ae38295c40c0009d6381fc4acb68fdf660
+python_version=3
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i CMakeLists.txt  -e '10aset(RUN_NEON "advanced")'
+	fi
+	vsed -i conf/CMakeLists.txt -e 's/\/etc\/udev\/rules.d\//\/usr\/lib\/udev\/rules.d/g'
```

Comment:
It's easier to use a separator other than `/` here... something like `s,things,things,`